fields/types/file/FileField.js

```diff
@@ -68,10 +68,13 @@ module.exports = Field.create({
 	hasExisting () {
 		return this.props.value && !!this.props.value.filename;
 	},
-	getFilename () {
+	getFilename (href) {
```

Comment:
It looks like there's another part of this missing - `href` isn't being passed in from the render method...